models/action/CancelCommittedOrder.model.bxb

```diff
@@ -2,22 +2,8 @@ action (CancelCommittedOrder) {
 
   type (CancelActivity)
 
-  confirmed-by (core.Confirmation) {
-    mode (PositiveEmphasis)
-    on-abort {
-      intent { //The intent to execute when user aborts the confirmation
-        goal {CheckStatus}
-      }
-    }
-    confirm-label {
-      if (exists(receipt)) {
-        template ("Cancel Order")
-      }
-    }
-
-    abort-label {
-      template ("Keep it")
-    }
+  confirm {
+    by (Confirmation)
```

Comment:
Confirmation concept is already being used for CommitOrder action. In my PR I made another concept called CancelConfirmation. Can one confirmation concept be used for two different actions and not have issues?

Reply:
Good comment. It's better to use different concepts if there will be different language. Right now I am only supporting simple yes/no and we have realized that using language other than yes/no is not a good practice in general. But I like to be safe and use different concepts just as you did. Thx.